### ISSUE TITLE
VCPKG fails to build mp3lame

### ISSUE THREAD

**Issue by xXTR0YXx** [AcademySoftwareFoundation/xstudio#107]
While trying to compile, VCPKG fails to compile MP3LAME with following error: 

[build-x64-windows-rel-out.log](https://github.com/user-attachments/files/15758276/build-x64-windows-rel-out.log)

**Comment by mpkepic** [comment]
Just to confirm, this also fails the same way when you try installing ffmpeg from vcpkg standalone, correct?

when installing Visual Studio, what languages were selected?  This looks like you are missing some core headers that should come with the compiler.

**Comment by xXTR0YXx** [comment]
no, ffmpeg builds fine. I have Visual Studio installed using english as language. Had issues with being set to german by default here in the past when trying to build the first time and if I remember right had issues when trying to build moonray
